src/utils/editor/source-documents.js

```diff
@@ -105,6 +107,23 @@ function showSourceText(editor: Object, source: Source) {
   updateLineNumberFormat(editor, source.id);
 }
 
+function resizeBreakpointGutter(editor: Object) {
+  const gutters = editor.display.gutters;
+  const breakpoints = gutters.querySelector(".breakpoints");
+  if (breakpoints) {
+    breakpoints.style.width = `${getLineNumberWidth(editor)}px`;
+  }
+}
+
+function resizeToggleButton(editor: Object) {
+  const toggleButton = document.querySelector(
+    ".source-header .toggle-button-start"
+  );
+  if (toggleButton) {
+    toggleButton.style.width = `${getLineNumberWidth(editor)}px`;
+  }
```

Comment:
I think this is he Ron module for these functions. This m is or code mirror documents. Let's move them to utils/editor/index

Reply:
sure

Reply:
@jasonLaster I will move the code to debugger utils. Will you take care of cleaning up the function in core modules? I will close my PR then?